### PR TITLE
Remove search_storage_objects.

### LINKED ISSUE #198: remove search_storage_objects method and supporting code (code for multiple moab instances)
`search_storage_objects method` and supporting code was added in PR #162 ;   this is no longer used and no longer needed, since PR sul-dlss/preservation_catalog/pull/1946 was merged.

### PR DESCRIPTION
**Pull request by justinlittman**
Hold pending https://github.com/sul-dlss/preservation_catalog/issues/1891

closes #198

## Why was this change made? 🤔
Deprecated

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact (e.g. changes what is written to filesystem, or what is expected to be read from filesystem), run ***[integration test create_preassembly_image_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test)*** on stage as it tests preservation, and/or test in stage environment, in addition to specs.⚡

### CODE REVIEW

**Review by ndushay — CHANGES_REQUESTED** [review]
There is no need to hold this PR;  the code that uses the method has already been removed from prescat.  

Could you please remove the fixture object "derivatives2" and trappings from the spec code if it's no longer being used?  See PR #162.

**Comment by justinlittman** [comment]
Thanks for the catch @ndushay. Fixed.

File: Gemfile.lock
Comment (ndushay) [inline Gemfile.lock:35]: did `Gemfile` change locally for you, Justin, or was the Gemfile.lock just out of date?  I'm ok with pry-byebug in either case;  just want to make sure the code base is consistent with itself.

**Review by ndushay — APPROVED** [review]
approving with one more question for consideration

File: Gemfile.lock
Comment (justinlittman) [reply Gemfile.lock:35]: I think the Gemfile.lock was out of date.

**Pull request merged**
